db/script.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+try {
+    $request = curl_init('https://dev.maydenacademy.co.uk/resources/store_products/products.json');
+    curl_setopt($request, CURLOPT_RETURNTRANSFER, 1);
+    $response = curl_exec($request);
+    if($response === false) {
+        throw new Exception('curl request failed');
+    }
+    $data = json_decode($response, true);
+    curl_close($request);
+
+    $db = new PDO('mysql:host=db;dbname=robot_stores', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+    $query = $db->prepare('DROP TABLE `products`');
```

Comment:
if there is no table of this name, mysql fails. you need to use `IF EXISTS` in all 3 drop table queries